src/PSRule.Rules.Azure/rules/Azure.Automation.Rule.ps1

```diff
@@ -32,27 +32,19 @@ Rule 'Azure.Automation.WebHookExpiry' -Ref 'AZR-000087' -Type 'Microsoft.Automat
 
 # Synopsis: Ensure automation account audit diagnostic logs are enabled.
 Rule 'Azure.Automation.AuditLogs' -Ref 'AZR-000088' -Type 'Microsoft.Automation/automationAccounts' -Tag @{ release = 'GA'; ruleSet = '2021_12'; } {
-    $diagnosticLogs = @(GetSubResources -ResourceType 'Microsoft.Insights/diagnosticSettings', 'Microsoft.Automation/automationAccounts/providers/diagnosticSettings');
-
-    $Assert.Greater($diagnosticLogs, '.', 0).Reason($LocalizedData.DiagnosticSettingsNotConfigured);
-
-    $logCategoryGroups = 'audit', 'allLogs';
-    $joinedLogCategoryGroups = $logCategoryGroups -join ', ';
-
-    foreach ($setting in $diagnosticLogs) {
-        $path = $setting._PSRule.path;
-        $auditLogs = @($setting.Properties.logs | Where-Object {
-            ($_.category -eq 'AuditEvent' -or $_.categoryGroup -in $logCategoryGroups) -and $_.enabled
-        });
-
-        $Assert.Greater($auditLogs, 'Length', 0).ReasonFrom(
-            'properties.logs',
-            $LocalizedData.AutomationAccountDiagnosticSetting,
-            $setting.name,
-            'AuditEvent',
-            $joinedLogCategoryGroups
-        ).PathPrefix($path);
-    }
+    $logCategoryGroups = 'audit', 'allLogs'
+    $joinedLogCategoryGroups = $logCategoryGroups -join ', '
+    $diagnostics = @(GetSubResources -ResourceType 'Microsoft.Insights/diagnosticSettings', 'Microsoft.Automation/automationAccounts/providers/diagnosticSettings' |
+        ForEach-Object { $_.Properties.logs |
+            Where-Object { ($_.category -eq 'AuditEvent' -or $_.categoryGroup -in $logCategoryGroups) -and $_.enabled }
+        })
+
+    $Assert.Greater($diagnostics, '.', 0).ReasonFrom(
+        'properties.logs',
```

Comment:
Because we are checking if the resources exists we want to report `resources` instead of `resources.properties.logs` which is not a valid path.

```suggestion
    $Assert.Greater($diagnostics, '.', 0).Reason(
```

Reply:
I understand. Will fix!